### DISCUSSION TITLE
Our end users lost all model permissions after a code build update.

### DISCUSSION THREAD

**Discussion by dmitrirussu** [spatie/laravel-permission#1995] [Q&A] [answered]
hi guys, at some point our "model has permissions" table became empty... could you advice why it happen. we do not have a UI to manage user permissions, once added no way to remove - delete them... but at some point after code release, we found that our table with "model has permissions" is empty and no user have access... have tried to find logs for but not found, our version is "spatie/laravel-permission:3.5.0", "Laravel Framework 6.20.27 ".  that specific table does not have soft delete in place, whad should be done to avoid such issue.

**Comment by erikn69** [comment, marked as answer]
No support for laravel-permission 3.x, only 5.x, feel free to make a PR to 3.x branch or upgrade your implementación
